src/layout/Header/LogInButton.tsx

```diff
@@ -1,17 +1,26 @@
-import styles from './Header.module.scss';
+export interface LogInButtonProps {
+  userName: string;
+  isSimplified: boolean;
+}
 
-export default function LogInButton({ userName }: { userName: string }) {
+export default function LogInButton({ userName, isSimplified }: LogInButtonProps) {
   return (
-    <div className={styles.account_label}>
-      <div className={styles.account_label__user_name_div}>
-        <div className={styles.account_label__first_letter}>{getFirstCapLetter(userName)}</div>
-        <div className={styles.account_label__user_name}> {userName}</div>
+    <div className="flex h-[40px] items-center gap-[30px]">
+      <div className="flex items-center gap-[10px]">
+        <div className="h-[40px] w-[40px] rounded-full bg-tk_cyanLight text-center font-Montserrat font-bold leading-[40px] text-tk_cyanDark">
+          {getFirstCapLetter(userName)}
+        </div>
+        {!isSimplified && <div className="p2">{userName}</div>}
       </div>
-      <div className={styles.account_label__sign_out}>Sign Out →</div>
+      {!isSimplified && <div className="p2 font-bold text-black">Sign Out →</div>}
     </div>
   );
 }
 
 function getFirstCapLetter(word: string): string {
   return word.charAt(0).toUpperCase();
 }
+
+LogInButton.defaultProps = {
+  isSimplified: false,
+};
```

Comment:
We dont use this format of default props anymore. This format normally uses in class components. If you need default props for any variable passed from its parent element, simply define a default on line 6.